srcpkgs/python3-pivy/template

```diff
@@ -13,18 +12,30 @@ license="0BSD"
 homepage="https://bitbucket.org/Coin3D/pivy"
 distfiles="https://bitbucket.org/Coin3D/pivy/get/${version}.tar.gz"
 checksum=2ee684dbf0f52c05b491c5423013fd82e679614bffee23641485248c8a790c4e
-nocross="require coin-config script"
 
 case "$XBPS_TARGET_MACHINE" in
 	ppc64*);;
 	ppc*) broken="{standard input}:3376761: Error: operand out of range";;
 esac
 
+if [ "$CROSS_BUILD" ]; then
+	CFLAGS+=" -I$XBPS_CROSS_BASE/usr/include/Coin3"
+fi
+
 post_extract() (
 	# use setup_old.py until have Coin with cmake support
 	mv -f setup_old.py setup.py
 )
 
+post_patch() {
+	if [ "$CROSS_BUILD" ]; then
+		vsed -i setup.py \
+			-e '/if not self[.]check_cmd_exists.*coin-config/,+1d' \
+			-e "s!coin-config!$XBPS_CROSS_BASE/usr/bin/coin-config!" \
+			-e "/INCLUDE_DIR.*coin-config/s!=!= \"$XBPS_CROSS_BASE\" +!"
```

Comment:
I think `coin-config` could be added to wrappers, this would simplify things. But it's good enough already.

Thanks @sgn